include/Capabilities/Hooks.php

```diff
@@ -0,0 +1,41 @@
+<?php
+/**
+ * @package Polylang
+ */
+
+namespace WP_Syntex\Polylang\Capabilities;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * A class allowing to determine if a user can translate content.
+ *
+ * @since 3.8
+ */
+class Hooks {
```

Comment:
```suggestion
class Retro_Compatibility {
```
I'm not very fond of the current class name 😅

Reply:
Well, I don't like `Retro_Compatibility` neither 😅 Moreover, this class has nothing to do with ~retro~ backward compatibility.